docs/README.md

```diff
@@ -26,9 +26,7 @@ contains the last release of documentation.
 
 After a release, documentation updates are continually merged into `master` as
 they occur. This work includes new documentation for forthcoming features, bug
-fixes, and other updates. Docker's CI system automatically builds and updates
-the `master` documentation after each merge and posts it to
-[http://docs.master.dockerproject.org](http://docs.master.dockerproject.org). 
+fixes, and other updates.
```

Comment:
Just wondering; Will this return in some form? I actually liked being able to point users to that to view upcoming docs changes